docs/content/singleton-actors.md

```diff
@@ -18,44 +18,23 @@ Due to Unreal server-workers spawning their own instances of each Singleton Acto
 ## Setting up Singleton Actors
 
 To properly set up Singleton Actors for your project, you need to:
-1. Register Singleton Actors with the `DefaultEditorSpatialGDK.ini` file
+1. Register Singleton Actors by tagging them with the `SpatialType=Singleton` class attribute.
 1. Add the generated components to the UnrealWorker worker configuration file
 
 ### Registering Singleton Actors
 
-You need to specify Singleton Actors in `DefaultEditorSpatialGDK.ini` before creating the snapshot and running SpatialOS.
+### Example code snippet
 
-Using the [Unreal GDK Starter Project](https://github.com/spatialos/UnrealGDKStarterProject) as an example, to do this:
-1. Locate the `DefaultEditorSpatialGDK.ini` file in the Starter Project repository which you cloned during installation and setup - it's located at `Game/Config/DefaultEditorSpatialGDK.ini`.
-2. Open the file in your editor and add your Singleton Actors as shown in the file snippet below.
-
-### Example file snippet
-
-In the snippet below, your game is `ExampleGame` and the Singleton Actor class you want to add is `ExampleGameGameState`
+The snippet below shows how to tag a class with the appropriate identifiers.
```

Comment:
Suggest:
1. Shifting this text under the header "##How to tag classes with Singleton Actor identifiers".
2. Adding "The code snippet below..."